apps/clinical/src/components/forms/medicationRequest/styles/MedicationRequestForm.module.scss

```
@@ -0,0 +1,42 @@
+@use "@carbon/react/scss/spacing" as *;
+@use "@carbon/react/scss/type" as *;
+@use "@carbon/react/scss/theme" as *;
+
+.form {
+  margin: $spacing-03;
+  background-color: $background;
+}
+
+.title {
+  @include type-style("heading-compact-02");
+  padding-bottom: $spacing-05;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)